code/game/machinery/sleepers.dm

```diff
@@ -98,35 +118,33 @@
 /obj/machinery/sleeper/MouseDrop_T(mob/target, mob/user)
 	if(HAS_TRAIT(user, TRAIT_UI_BLOCKED) || !Adjacent(user) || !user.Adjacent(target) || !iscarbon(target) || !ISADVANCEDTOOLUSER(user))
 		return
-
 	close_machine(target)
 
-
 /obj/machinery/sleeper/screwdriver_act(mob/living/user, obj/item/I)
-	. = TRUE
-	if(..())
-		return
+	. = ..()
```

Comment:
you removed the `if (.)` check why

Reply:
Parent is just a return, there was no point in having it.